### DISCUSSION TITLE
リソースグループを日本語で入力したときにVSCodeからアクセスができない

### DISCUSSION THREAD

**Discussion by momora-m** [1ft-seabass/ms-azure-hackathon-league-2021#46] [Show and tell]
# 日本語でリソースグループを作成したのち、VSCodeにAzureの拡張を導入してアクセスしようとすると以下の画像とメッセージが表示されてしまい、配下のリソースにアクセスできない

![image](https://user-images.githubusercontent.com/36496292/146631346-3a16c9b9-2fb2-4442-8de5-fdf78006bd05.png)

### エラー文言
*"resourceGroupName" with value "ハッカソン" should satisfy the constraint "Pattern": /^[-\w\._\(\)]+$/.*

### 備考
リソースグループが英字の場合は配下のリソースは問題なく確認できる

**Comment by darkcrash** [comment]
私も過去に、日本人らしくすべての日本語でデプロイしようと試みましたが、Validationを通過させてたとしても
その後にエラーとなったり、削除できなくなって、サポートに削除してもらう結果になったことがありました。
時間が迫っている場合は、アルファベットで名称をつけていくほうが無難ではあります。

**Reply by momora-m** [reply]
ありがとうございます。英語名にして対処するようにしたいと思います。
